README.md

```diff
@@ -19,7 +19,7 @@ $ "$GOPATH/bin/wuzz" --help
 ### Commands
 
 Keybinding                              | Description
-----------------------------------------|------------------------------------------------------------
+----------------------------------------|---------------------------------------
```

Comment:
Yeah, that was just a fix to keep the README at 79cols. I don't think it need a commit on its own, but i can split it if needed